cmd/build/helmify/kustomize-for-helm.yaml

```diff
@@ -80,6 +80,7 @@ spec:
             - --exempt-namespace={{ .Release.Namespace }}
             - --operation=webhook
             - --enable-external-data={{ .Values.enableExternalData }}
+            - --enable-generator-resource-expansion={{ .Values.enableGeneratorResourceExpansion }}
```

Comment:
This needs to be added for the audit deployment below.

Reply:
@ritazh fix LGTY?